Samples/WindowsAppSDKTelemetry/cpp/TelemetrySample.cpp

```diff
@@ -0,0 +1,11 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+#include "TelemetrySample.h"
+
+int main()
+{
+    WindowsAppSDKTelemetrySample::BooleanTelemetryEvent(true);

```

Comment:
Can you please provide examples of how partC, PartB looks like for these examples?